cabal-install/cabal-install.cabal

```diff
@@ -214,12 +217,15 @@ executable cabal
     -- NOTE: when updating build-depends, don't forget to update version regexps
     -- in bootstrap.sh.
     build-depends:
+        async      >= 2.0      && < 3,
         array      >= 0.4      && < 0.6,
         base       >= 4.5      && < 5,
         binary     >= 0.5      && < 0.9,
+        byteable   >= 0.1      && < 0.2,
         bytestring >= 0.9      && < 1,
         Cabal      >= 1.24     && < 1.25,
         containers >= 0.4      && < 0.6,
+        cryptohash >= 0.11     && < 0.12,
```

Comment:
@dcoutts BTW, `cryptohash` has been deprecated in favour of `cryptonite`, and versions starting with 0.11.8 pull in the whole of `cryptonite`. So we probably should set the upper bound for `cryptohash` at 0.11.8.